grafana/dashboards/cmode/security.json

```diff
@@ -84,1140 +84,3865 @@
-        "w": 5,
-        "x": 10,
-        "y": 1
-      },
-      "id": 210,
-      "options": {
-        "colorMode": "value",
-        "graphMode": "area",
-        "justifyMode": "auto",
-        "orientation": "auto",
-        "reduceOptions": {
-          "calcs": [
-            "lastNotNull"
-          ],
-          "fields": "",
-          "values": false
-        },
-        "text": {},
-        "textMode": "auto"
-      },
-      "pluginVersion": "8.3.4",
-      "targets": [
-        {
-          "datasource": "${DS_PROMETHEUS}",
-          "exemplar": false,
-          "expr": "((count(svm_labels{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\", anti_ransomware_state!=\"\", anti_ransomware_state=~\"enabled|dry_run\"}) or vector (0))\n/\n(count(svm_labels{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\", anti_ransomware_state!=\"\"}) or vector (1))) * 100",
-          "instant": true,
-          "interval": "",
-          "legendFormat": "",
-          "refId": "A"
-        }
-      ],
-      "title": "SVM Anti-ransomware Status %",
-      "transformations": [
-        {
-          "id": "calculateField",
-          "options": {
-            "binary": {
-              "left": "encrypted",
-              "operator": "/",
-              "reducer": "sum",
-              "right": "total"
-            },
-            "mode": "binary",
-            "reduce": {
-              "reducer": "sum"
-            },
-            "replaceFields": true
-          }
-        },
-        {
-          "id": "calculateField",
-          "options": {
-            "binary": {
-              "left": "encrypted / total",
-              "operator": "*",
-              "reducer": "sum",
-              "right": "100"
-            },
-            "mode": "binary",
-            "reduce": {
-              "reducer": "sum"
-            },
-            "replaceFields": true
-          }
-        }
-      ],
-      "type": "stat"
-    },
-    {
-      "datasource": "${DS_PROMETHEUS}",
-      "description": "",
-      "gridPos": {
-        "h": 1,
-        "w": 9,
-        "x": 15,
-        "y": 1
-      },
-      "id": 201,
-      "options": {
-        "content": "",
-        "mode": "markdown"
-      },
-      "pluginVersion": "8.3.4",
-      "title": "Cluster Authentication Methods",
-      "type": "text"
-    },
-    {
-      "datasource": "${DS_PROMETHEUS}",
-      "description": "",
-      "gridPos": {
-        "h": 1,
-        "w": 6,
-        "x": 15,
-        "y": 2
-      },
-      "id": 211,
+      "id": 213,
       "options": {
-        "content": "",
+        "content": "View ONTAP Security Hardening Guide https://www.netapp.com/media/10674-tr4569.pdf for more details.",
         "mode": "markdown"
       },
-      "pluginVersion": "8.3.4",
-      "title": "Authentication Methods",
+      "pluginVersion": "8.1.8",
+      "title": "Important Information about Compliance dashboard",
       "type": "text"
     },
     {
+      "collapsed": true,
```

Comment:
Probably better to show `Highlights` when dashboard is loaded

Reply:
corrected.